src/platform/user/authorization/components/IdentityNotVerified.jsx

```diff
@@ -20,10 +20,7 @@ const VerifyIdentityInfo = () => (
     trigger="If you have trouble verifying your identity"
     uswds
   >
-    <a
-      href="https://www.va.gov/resources/verifying-your-identity-on-vagov/"
-      clasName="vads-u-padding-bottom--2"
```

Comment:
@department-of-veterans-affairs/octo-identity Removing `clasName` here since it generates a warning in the console, and the resulting UI passed previous code review.

Reply:
@radavis Can you just fix property name? `clasName` => `className`?

Reply:
I'll remove this change and fix in a separate PR.

Reply:
No need. My team can take this one. Thank you for bringing it to my attention.